### DISCUSSION TITLE
Problem with permession user

### DISCUSSION THREAD

**Discussion by therus000** [strimzi/strimzi-kafka-operator#10658] [Q&A]
Good day 
i got this in error in logs my kafka broker

"""
2024-10-01 12:30:39,228 INFO Principal = User:writer is Denied Operation = Read from host = 192.168.1.55 on resource = Group:LITERAL: for request = Heartbeat with resourceRefCount = 1 (kafka.authorizer.logger) [data-plane-kafka-request-handler-2]
"""

i tried to add this permission for this user
but it is not helped

      - resource:
          type: group
          name: heartbeat
          patternType: literal
        operation: Read 
        host: "*"    

maybe someone can help with this error?

**Comment by scholzj** [comment]
What is the application that triggers it? How is it configured? It seems to send heartbeat requests for a group that has no name (`Group:LITERAL:`)? Or maybe the name is empty String? That does not seem valid. You should check the client and see if the group is properly configured. In any case, I would not expect your ACL to work because the group name is not `heartbeat`, that is the request that was denied.

**Reply by therus000** [reply]
thnx. ill try to check the service that causes this error
